src/spans/awsSpan.test.js

```diff
@@ -622,7 +619,78 @@ describe('awsSpan', () => {
       version: '$LATEST',
     };
 
-    expect(awsSpan.getHttpSpan(requestData, responseData)).toEqual(expected);
+    const result = awsSpan.getHttpSpan(id, requestData, responseData);
+    expect(result).toEqual(expected);
+  });
+
+  test('getHttpSpan - only for request data', () => {
+    const id = 'not-a-random-id';
+    const sendTime = 1234;
+
+    const requestData = {
+      host: 'your.mind.com',
+      headers: { Tyler: 'Durden' },
+      body: 'the first rule of fight club',
+      sendTime,
+    };
+    const expected = {
+      account: '985323015126',
+      ended: undefined,
+      id: 'not-a-random-id',
+      info: {
+        httpInfo: {
+          host: 'your.mind.com',
+          request: {
+            body: '"the first rule of fight club"',
+            headers: '{"Tyler":"Durden"}',
+            host: 'your.mind.com',
+            sendTime: 1234,
+          },
+          response: {},
+        },
+        logGroupName: '/aws/lambda/aws-nodejs-dev-hello',
+        logStreamName: '2019/05/16/[$LATEST]8bcc747eb4ff4897bf6eba48797c0d73',
+        traceId: {
+          Parent: '28effe37598bb622',
+          Root: '1-5cdcf03a-64a1b06067c2100c52e51ef4',
+          Sampled: '0',
+          transactionId: '64a1b06067c2100c52e51ef4',
+        },
+        tracer: {
+          name: '@lumigo/tracerMock',
+          version: '1.2.3',
+        },
+      },
+      memoryAllocated: '1024',
```

Comment:
Many of these keys are coming from `SpanBuilder`. Do you think they should be part of the test?

Reply:
I agree in general, this test specify checks the build of the spans, I don't want to hide bug under our builder